ietf/templates/iab_base.html

```diff
@@ -52,7 +52,7 @@
         {% analytical_head_bottom %}
     <link href="{% static 'dist/main.css' %}" rel="stylesheet"></head>
 
-    <body class="{% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}">
+    <body class="{% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %} iab_body">
```

Comment:
I think this file will get replaced (auto-created) on deployment.
See the warning on on line 6.

Reply:
That warning doesn't apply to this file, so I've removed it. It should all work - this template doesn't have a corresponding file in `templates_src` to overwrite it.